app/javascript/courses/review/components/CoursesReview__ChecklistEditor.res

```diff
@@ -182,12 +195,39 @@ let make = (~reviewChecklist, ~updateReviewChecklistCB, ~closeEditModeCB, ~targe
                       active={invalidTitle(ReviewChecklistItem.title(reviewChecklistItem))}
                     />
                   </div>
-                  <button
-                    title={t("checklist_title.remove_button_title")}
-                    className="bg-gray-200 p-2 w-11 border border-gray-400 text-gray-700 rounded ml-2 hover:text-red-600 hover:bg-red-100 focus:outline-none"
-                    onClick={_ => removeChecklistItem(itemIndex, setState)}>
-                    <i className="fas fa-trash-alt" />
-                  </button>
+                  <div
+                    className="-mr-10 flex-shrink-0 border bg-gray-100 rounded-lg flex flex-col text-xs sticky top-0">
+                    {controlIcon(
+                      ~icon="fa-arrow-up",
+                      ~title={t("checklist_title.move_up_button_title")},
+                      ~handler={
+                        _ =>
+                          setState(state => {
+                            ...state,
+                            reviewChecklist: ArrayUtils.swapUp(itemIndex, state.reviewChecklist),
+                          })
+                      },
```

Comment:
Using an action reducer could make this much more readable.
```suggestion
                      ~handler={ _=>send(SwapUpReviewChecklist(itemIndex) }
```

Reply:
Done